crates/node_binding/src/js_values/stats.rs

```diff
@@ -377,6 +469,16 @@ impl JsStats {
       .collect()
   }
 
+  #[napi]
+  pub fn get_logging(&self) -> Vec<JsStatsLogging> {
```

Comment:
Logging is hard to distinguish from "compilationLogging", maybe we could name something more specific.
```
pub fn get_js_stats_logging(&self) -> Vec<JsStatsLogging>;
// or
pub fn get_stats_logging(&self) -> Vec<JsStatsLogging>;
```

Reply:
It's easy to distinguish them by the function signature, and logging basically only consumed by stats, so specific `get_logging` to `get_stats_logging` seems not necessary